task/backend/analytical_storage_test.go

```diff
@@ -64,7 +64,7 @@ func TestAnalyticalStore(t *testing.T) {
 				svcStack = backend.NewAnalyticalRunStorage(logger, svc, ts.BucketService, svc, rr, ab.QueryService())
 			)
 
-			ts.BucketService = storage.NewBucketService(ts.BucketService, ab.storageEngine)
+			ts.BucketService = storage.NewBucketService(nil, ts.BucketService, ab.storageEngine)
```

Comment:
Any concerns that this `nil` might blow up on us later?

Reply:
Absolutely – pushed a fix and thanks for finding it. My IDE hid that reference from me :)